src/index.js

```diff
@@ -1,6 +1,18 @@
 import './style.css';
-import UI from './modules/display.js';
+import ApiOp from './modules/display.js';
+
+const refereshButton = document.getElementById('refresh');
```

Comment:
[OPTIONAL] Since HTML&CSS Best practices is not a part of this project requirement I will keep it optional. Kindly fix typo error in your files `action="submite"`to `action="submit"` in line:23 in HTML file as the previous reviewer requested and here we have another one, it is one of the requirements of this project to follow [JavaScript Best Practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md). Kindly fix refereshButton to refreshButton for better naming 👍

Reply:
@huseyinbiyik  Thank you for your thorough review 👍 .
I implemented the required changes as recommended 😃 .